package.json

```diff
@@ -42,8 +42,8 @@
     "esm": "^3.2.25",
     "fast-check": "^2.0.0",
     "prettier": "^1.19.1",
-    "ts-node": "^8.10.2",
-    "typescript": "^3.9.6"
+    "ts-node": "^9.0.0",
+    "typescript": "^4.0.3"
```

Comment:
No, the change I made does not require the version bump. 

Since this is a `devDependency`, is this change even exposed to consumers of `ava-fast-check`? I believe since the code is compiled into JS with this dependency before publication to NPM, there is no effect on downstream users and we wouldn't need to call this a major version bump.

Reply:
Not expose to our users, my point is more that if we compile against TS 4.0 our d.ts might declare typings not compatible with TS <4.0. It just that TS does not offer a great tooling (at the moment) to ensure that your package keeps compatible with old releases.